erpnext/erpnext_integrations/connectors/woocommerce_connection.py

```diff
@@ -20,45 +24,44 @@ def verify_request():
 		not sig == bytes(frappe.get_request_header("X-Wc-Webhook-Signature").encode()):
 			frappe.throw(_("Unverified Webhook Data"))
 	frappe.set_user(woocommerce_settings.creation_user)
+	print ("Request verified successfully")
```

Comment:
print

Reply:
Should I edit the page here:
https://erpnext.com/docs/user/manual/en/erpnext_integration/woocommerce_integration
?